airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/ConfigsFlywayMigrationDatabase.java

```diff
@@ -14,10 +14,6 @@
  */
 public class ConfigsFlywayMigrationDatabase extends FlywayMigrationDatabase {
 
-  public ConfigsFlywayMigrationDatabase() {
```

Comment:
Unused constructor, was confusing regarding why it is accepting the schema_dump file

Reply:
this isn't used by flyway somehow? if i'm wrong ignore it. @tuliren

Reply:
Yes, confirmed that this constructor is no longer used.